features/product/viewing_products/viewing_product.feature

```diff
@@ -0,0 +1,14 @@
+@viewing_products
+Feature: Viewing a product details
+    In order to see products detailed information
+    As a Visitor
+    I want to be able to view a single product
+
+    Background:
+        Given the store operates on a single channel in "France"
+
+    @ui
+    Scenario: Viewing a detailed page with product's name
+        Given the store has a product "T-shirt banana"
+        When I open this product page
```

Comment:
We should avoid using UI specific naming, unless scenario is UI-only, like permalinks stuff.